runtimes/common/src/xcm_config.rs

```diff
@@ -69,7 +69,7 @@ impl ShouldExecute for DenyReserveTransferToRelayChain {
 		instructions: &mut [Instruction<Call>],
 		_max_weight: Weight,
 		_weight_credit: &mut Weight,
-	) -> Result<(), ()> {
+	) -> Result<(), ProcessMessageError> {
```

Comment:
Can you replace this implementation of the barrier with the new matcher? https://github.com/paritytech/cumulus/blob/f603a61ff370fc33740c9373833c3c6ba1486846/parachains/common/src/xcm_config.rs#LL40C12-L40C43

Reply:
I added a comment which points to the new xcm matcher PR [5abb95b](https://github.com/KILTprotocol/kilt-node/pull/527/commits/5abb95b54ab65ee280e8ecf1986e80321351cee5).  Did you mean that?